src/main/resources/jsonSchemas/antoraComponentSchema.json

```diff
@@ -20,13 +19,22 @@
       "type": [
         "string",
         "number",
+        "boolean",
+        "object",
         "null"
       ],
       "description": "The version key is the version coordinate in the resource ID, version segment in the page URL, and, when display_version isn’t set, the version name displayed in the UI. It’s also used to sort multiple versions of the same component. Use '~' (without quotes) for a versionless component. Examples: '1', 'v2.0-beta', 'rawhide'",
```

Comment:
Not sure how well we can replace https://docs.antora.org/antora/latest/component-version-key/ in a short description. Maybe include a link instead?

Reply:
Not sure if they are support in the IDE. Please give it a try; if they are, feel free to add a link.

Reply:
I'll admit I haven't set up a local build, sorry. :sweat_smile: 

Not sure what IDEA's JSON Schema support resp. it's Quick Help support. If there's need, we could request features from them?